src/em/em.cpp

```diff
@@ -668,6 +668,17 @@ int em_t::send_frame(unsigned char *buff, unsigned int len, bool multicast)
         }
     }
 #ifdef AL_SAP
+#ifdef DEBUG_MODE
+    printf("PP_DEBUG_ORIGINAL_ETH_FRAME:\t");
+
+    for(int i = 0 ; i < len; ++i){
+        printf(" %02x ",buff[i]);
+    }
+    printf("\n");
```

Comment:
I'd recommend using `util::print_hex_dump`, it's already included in `util.h`

It can also print to module log files but the default to `stdout`.

```suggestion
    util::print_hex_dump(len, buff);
```